content/docs/deployment/csminstallationwizard/_index.md

```diff
@@ -104,13 +106,13 @@ The [Dell Container Storage Modules Installation Wizard](./src/index.html) is a
 
 2. Look over all the fields in the generated `values.yaml` and fill in/adjust any as needed.
 
-3. For the Observability module, please refer [Observability](../csmoperator/modules/observability) to install the post installation dependencies.
+3. If Observability is enabled, please refer to [Observability](../csmoperator/modules/observability) to install the post installation dependencies.
```

Comment:
change to  "If Observability is enabled, please refer to [Observability](../csmoperator/modules/observability) for post installation dependencies"